deploy/customResourceDefinitions.yaml

```diff
@@ -41,3 +53,27 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
+  additionalPrinterColumns:
```

Comment:
glad to see you're not bothering with the unused obc fields, Versioned, SSL, etc.

Reply:
I am bothered that it is not generated the go structures.